src/appkit.cs

```diff
@@ -15941,6 +15968,7 @@ partial interface NSView : NSDraggingDestination, NSAnimatablePropertyContainer,
 		[Export ("drawPageBorderWithSize:")]
 		void DrawPageBorder (CGSize borderSize);
 		
+		[Deprecated (PlatformName.MacOSX, 10, 14, message: "Never called.")]
```

Comment:
What do you mean never called?

Reply:
The note from apple said this method is never called.

Reply:
That's not exactly true, you can implement NSView and override various Draw functions and AppKit will call them. 

I could say something like "This method is never called and does nothing", but that feels like a mouthful.